src/Concerns/ManagesTransactions.php

```diff
@@ -168,7 +168,7 @@ protected function performRollBack($toLevel)
 
         if ($this->currentTransaction !== null) {
             try {
-                if ($this->currentTransaction->state() === Transaction::STATE_ACTIVE) {
+                if ($this->currentTransaction->state() === Transaction::STATE_ACTIVE && $this->currentTransaction->id() !== null) {
```

Comment:
When rolling back a transaction, an exception is now thrown if the target transactionId is not specified.

ref: https://github.com/googleapis/google-cloud-php/pull/6708/files#diff-25ee9adab31aa54b734a2bc5f254b0e922b13e1c0ad27f1fff60a9a534552a03
L192-L194

This caused the following tests to fail.

```
1) Colopl\Spanner\Tests\ConnectionTest::test_connection_transaction_reset_on_exceptions
InvalidArgumentException: Rollback failed: Transaction not initiated.

/project/vendor/google/cloud-spanner/src/Operation.php:193
/project/vendor/google/cloud-spanner/src/Transaction.php:572
/project/src/Concerns/ManagesTransactions.php:172
/project/vendor/laravel/framework/src/Illuminate/Database/Concerns/ManagesTransactions.php:264
/project/src/Concerns/ManagesTransactions.php:73
/project/vendor/google/cloud-spanner/src/Database.php:957
/project/vendor/google/cloud-core/src/Retry.php:80
/project/vendor/google/cloud-spanner/src/Database.php:975
/project/src/Concerns/ManagesTransactions.php:76
/project/src/Connection.php:525
/project/src/Concerns/ManagesTransactions.php:81
/project/tests/ConnectionTest.php:541
```

Originally, when a transaction was called without specifying any options, `Operation::beginTransaction()` was called internally to set the transactionId.
With this change, the default options specified have been changed and `Operation::beginTransaction()` is no longer called internally.

ref: https://github.com/googleapis/google-cloud-php/pull/6708/files#diff-25ee9adab31aa54b734a2bc5f254b0e922b13e1c0ad27f1fff60a9a534552a03
L495-L497

As a result, `InvalidArgumentException('Rollback failed: Transaction not initiated.')` exception is thrown when rollback is done without calling any query in `Database::runTransaction()`.